doc/under_sampling.rst

```diff
@@ -6,8 +6,25 @@ Under-sampling
 
 .. currentmodule:: imblearn.under_sampling
 
-You can refer to
-:ref:`sphx_glr_auto_examples_under-sampling_plot_comparison_under_sampling.py`.
+One way of handling imbalanced datasets is to reduce the number of observations from
+the majority class or classes. The most well known algorithm in this group is random
```

Comment:
> the majority class or classes

I think it is a bit weird to say majority classes since usually we would refer to a single class. What about saying something like:

"reduce the number of observations from all classes but the minority class (i.e. the one with the least number of observations)."